src/main/java/core/basesyntax/fruitstoreoperation/BuyFruitOperation.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.fruitstoreoperation;
+
+import core.basesyntax.FruitStoreDao;
+import core.basesyntax.InputDataModel;
+import core.basesyntax.exception.NoSuchFruitException;
+import core.basesyntax.exception.NotEnoughFruitException;
+import java.time.LocalDate;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class BuyFruitOperation implements FruitStoreOperation {
+    private FruitStoreDao fruitStoreDao = new FruitStoreDao();
```

Comment:
better use constructor of the class for this
we will learn DI later

Reply:
done